hiddenlayer/sdk/rest/api_client.py

```diff
@@ -1,7 +1,7 @@
 # coding: utf-8
 
 """
-    HiddenLayer ModelScan
+    HiddenLayer ModelScan V2
```

Comment:
V3?

Reply:
Ultimately, this is what the generator created.  I'll take a look at the OAI specs we're using, but I am using a combination of V2 and V3 endpoints.

Reply:
Confirmed, this is happening because I'm merging multiple OpenAPI specs and the first one in the list has its title as HiddenLayer ModelScan V2.  Given that this is just a comment in the client, I'm inclined to leave it alone.